sys/net/application_layer/dhcpv6/client.c

```diff
@@ -185,6 +230,27 @@ void dhcpv6_client_req_ia_pd(unsigned netif, unsigned pfx_len)
     }
 }
 
+void dhcpv6_client_req_ia_na(unsigned netif)
```

Comment:
Could make sense to return `-ENOMEM`, when there is no lease entry available anymore.